### PR TITLE
Create participants.md file

### PR DESCRIPTION
**Pull request by rahmot**
Hi! This is the `participants.md` file, I will link it to the [navbar](https://github.com/bebatut/galaxy_mentor_network/pull/20)

### CODE REVIEW

File: content/networks.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+id: Networks
```
Comment (bebatut) [inline content/networks.md:2]: Why a "s" in the id there? I have the feeling it is one network of mentors, mentees and network managers

File: content/networks.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+title: Meet Our Participants
```
Comment (bebatut) [inline content/networks.md:3]: I am not sure if the Title should not be the same as in the navbar. Maybe we could have this sentence as a subtitle

File: content/networks.md (hunk 1 above)
Comment (rahmot) [reply content/networks.md]: Thanks for pointing! I have removed it.

File: content/networks.md (hunk 2 above)
Comment (rahmot) [reply content/networks.md]: Done, thanks!

File: content/network.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+id: Network
```
Comment (bebatut) [inline content/network.md:2]:
```suggestion
id: network
```

**Comment by bebatut** [comment]
It looks good. I merge. Thanks

**Pull request merged**